Content.Server/_DV/AACTablet/AACTabletSystem.cs

```diff
@@ -0,0 +1,60 @@
+using Content.Server.Chat.Systems;
+using Content.Server.Speech.Components;
+using Content.Shared._DV.AACTablet;
+using Content.Shared.IdentityManagement;
+using Robust.Shared.Prototypes;
+using Robust.Shared.Timing;
+
+namespace Content.Server._DV.AACTablet;
+
+public sealed class AACTabletSystem : EntitySystem
+{
+    [Dependency] private readonly ChatSystem _chat = default!;
+    [Dependency] private readonly IGameTiming _timing = default!;
+    [Dependency] private readonly IPrototypeManager _prototype = default!;
+
+    private readonly List<string> _localisedPhrases = [];
+
+    public const int MaxPhrases = 10; // no writing novels
+
+    public override void Initialize()
+    {
+        base.Initialize();
+        SubscribeLocalEvent<AACTabletComponent, AACTabletSendPhraseMessage>(OnSendPhrase);
+    }
+
+    private void OnSendPhrase(Entity<AACTabletComponent> ent, ref AACTabletSendPhraseMessage message)
+    {
+        if (ent.Comp.NextPhrase > _timing.CurTime || message.PhraseIds.Count > MaxPhrases)
+            return;
+
+        var senderName = Identity.Entity(message.Actor, EntityManager);
+        var speakerName = Loc.GetString("speech-name-relay",
+            ("speaker", Name(ent)),
+            ("originalName", senderName));
+
+        _localisedPhrases.Clear();
+        foreach (var phraseProto in message.PhraseIds)
+        {
+            if (_prototype.TryIndex(phraseProto, out var phrase))
+            {
+                // Ensures each phrase is capitalised to maintain common AAC styling
```

Comment:
i was going to complain about every phrase being capitalised in a built string but. okay fine i guess